### DISCUSSION TITLE
Welcome to FunASR Discussions!

### DISCUSSION THREAD

**Discussion by LauraGPT** [modelscope/FunASR#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xdlin424** [comment]
前端接收到的语音识别结果，断句为何是以上一句标点开头
比如 "text\":\"，不得喧哗吵闹，不得录音录像摄影\"

**Comment by dvjsbdufbv** [comment]
File "/opt/anaconda3/envs/funasr/lib/python3.10/site-packages/torch/serialization.py", line 1338, in _legacy_load
    magic_number = pickle_module.load(f, **pickle_load_args)
_pickle.UnpicklingError: invalid load key, 'v'.

**Comment by Stephanie-ting** [comment]
FunASR-nano-2512 + vLLM的AutoModelVLLM是否可以支持输出sentence_info句子级别的时间戳，配置了vad_model、spk_model和sentence_timestamp=true仍只输出token字级别的时间戳
